package.json

```diff
@@ -73,6 +73,6 @@
     "path-to-regexp": "^1.7.0",
     "swagger-methods": "^1.0.0",
     "swagger-schema-official": "2.0.0-bab6bed",
-    "z-schema": "^3.22.0"
+    "z-schema": "^5.0.1"
```

Comment:
Want to add this to the changelog you created and also bump the module version to 2.1.1?

Reply:
done